src/clients/import/ImportForm.tsx

```diff
@@ -74,7 +110,13 @@ export default function ImportForm() {
           role="manage-clients"
         >
           <FormProvider {...form}>
-            <JsonFileUpload id="realm-file" onChange={handleFileChange} />
+            <FileUploadForm
+              id="realm-file"
+              language={Language.json}
+              extension=".json"
```

Comment:
Should XML not also be allowed here?
```suggestion
              extension=".json,.xml"
```